### PR TITLE
Compatibility for ST2 and ST3

### PR DESCRIPTION
**Pull request by amullins83**
This follows the same strategy as used to make Rake compatible: a master python package file checks the version of python in use, then imports the appropriate plugins. The readme has been updated to remove the "2". This will require a small change to the package_control_channel repository so that Ceedling is visible to Package Control in Sublime Text 3.

### CODE REVIEW

File: python2/OpenCeedlingFile.py [hunk 1]
```diff
@@ -1,71 +1,71 @@
...
+			config_matcher = re.compile(r"(\\\\)?(project|test)\.yml$")
+			self.open_project_file(config_matcher, self.window)
+		
+		elif re.search(r"\w+\.[ch]\w*$", current_file_path):
+			
+			current_file = re.search(r"([\w\.]+)$", current_file_path).group(1)
+			base_name = re.search(r"(\w+)\.(\w+)$", current_file).group(1)
+			base_name = re.sub('^test_', '', base_name)
+
+			print "Basename: " + base_name
```
Comment (bdun) [inline python2/OpenCeedlingFile.py:25]: Is it cleaner to duplicate all the code for py2 and py3?  Or would it be better to just change the parts that are not py3 compatible like these prints to add braces?

**Review by bdun — APPROVED** [review]
This code looks great to have in.  I'm keen to use in ST3.

**Comment by FichteFoll** [comment]
Since the package seems abandoned, I'd be willing to merge PRs, but this PR in particular didn't age too well.First, it's pretty easy to write Python code that runs in both 2 and 3 runtimes without modifications and that's by far preferred over duplicating almost the entire code. Second, less than 0.5% of ST users are on ST2 according to Package Control statistics and it makes little sense to limit yourself to not only Python 2.6 but also to an old API that has seen several incremental improvements.

If someone were to port the current plugin to Python 3 and ST3, I would merge it and apply the necessary changes in package_control_channel.

**Comment by FichteFoll** [comment]
Superseded by #6.

**Pull request closed without merging**